### DISCUSSION TITLE
請問方便推廣此中文化嗎

### DISCUSSION THREAD

**Discussion by lktb2000** [GpointChen/FFXIVChnTextPatch-GP#107] [General]
為了能讓更多人享受FFXIV的樂趣
請問方便使用YT影片，推廣此中文化嗎
也會一併告知低調使用原則
並且只會提到有這個Github的存在
不會直接貼上該Github網址

**Comment by GpointChen** [comment]
原則上可以，請自便。

**Comment by lktb2000** [comment]
好的，非常感謝~ :)

**Comment by cvsh02060** [comment]
請問有接受贊助的嗎? 感謝大大讓我們有6.0繁中可以玩

**Comment by DirBreeze** [comment]
> 請問有接受贊助的嗎? 感謝大大讓我們有6.0繁中可以玩

真的，想獻上自己的小小心意，也讓大大辛苦付出的時間能有一些回報

**Comment by Bally0886** [comment]
+1, 真心感謝大大的幫忙。

**Comment by HaseoCorleone** [comment]
> 為了能讓更多人享受FFXIV的樂趣 請問方便使用YT影片，推廣此中文化嗎 也會一併告知低調使用原則 並且只會提到有這個Github的存在 不會直接貼上該Github網址


上一個漢化被人用在直播上的原作者下場你不知道嗎？要推廣就介紹給實際認識的人就好了，為何還要發影片出來？6.0改版熱潮才過多久，會想玩的早就在玩了還需要再推廣？拜託不要來亂，你就那麼相信網上的其他人會低調啊！到時候版主被官方關切，又害大家沒得用你是能負責嗎？
